### PR TITLE
Add link to dbt nux in dbt integration page

### PR DESCRIPTION
**Pull request by clairelin135**
Adds links to the dbt nux to the dbt + dagster integration page: https://docs.dagster.io/integrations/dbt

### CODE REVIEW

**Comment by clairelin135** [comment]
Current dependencies on/for this PR:
* master
  * **PR #16885** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/16885" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/16885?utm_source=stack-comment).

**Comment by clairelin135** [comment]
@sryza @erinkcochran87 could I get a review on this when you have the chance?

File: docs/content/integrations/dbt.mdx [hunk 1]
```diff
@@ -5,6 +5,13 @@ description: Dagster can orchestrate dbt alongside other technologies.
 
 # dbt + Dagster
 
+<Note>
+  Automatically load your dbt models as Dagster assets by{" "}
+  <a href="https://dagster.cloud/signup?next=/prod/getting-started%3Ftab%3Dimport_dbt_core_project%26serverless%3D1">
```
[the remark this answers is not in the document]
Comment (clairelin135) [reply docs/content/integrations/dbt.mdx]: yup, added

**Review by erinkcochran87 — APPROVED** [review]

**Review by sryza — APPROVED** [review]

**Review by sryza — CHANGES_REQUESTED** [review]
Sorry for the approve-then-request changes, but thinking about this, I think we could be a little more clear about how we present this. I'm worried that readers of this page will see the banner at the top and assume this is the only way for Dagster to work with dbt.

This leads me to think that maybe we should omit it at the top, and instead find a way to make it more prominent within the "Getting started" section.

**Comment by erinkcochran87** [comment]
If you want to keep the callout, we could also update the copy - ex:

**Using Dagster Cloud?** You can automatically load your dbt models as Dagster assets by importing a dbt project in Dagster Cloud.

Definitely not a hill I'll die on tho

**Comment by clairelin135** [comment]
@erinkcochran87's suggestion above makes sense to me, I updated this PR.

@sryza let me know if this still looks confusing

**Review by sryza — APPROVED** [review]
I think this is much better

**Pull request merged**